runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -229,6 +231,21 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 	}
+	
+	@Subscribe
+	public void loadWidgets(WidgetLoaded event)
+	{
+		if (!config.screenshotKingdom())
+		{
+			return;
+		}
+		int widget = WidgetID.KINGDOM_GROUP_ID;
```

Comment:
Unused variable.

Reply:
Yes and it didn't trigger. Also that variable was for testing. I thought I removed it, my mistake.